lib/decorators/google-pubsub-delivery-attempts.decorator.ts

```diff
@@ -0,0 +1,12 @@
+import { Message } from '@google-cloud/pubsub';
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
+import { GooglePubSubContext } from '../ctx-host';
+
+export const getDeliveryAttempts = (key: string | undefined, ctx: ExecutionContext): number => {
```

Comment:
What about exposing the whole message? Just thinking if you need to expose more in the future you won't have to create another decorator for exposing each part of the message.

Reply:
That gives a little too much power since the user would also get access to the raw `ack`/`nack` handlers which could cause some havoc.

Reply:
ok, fair enough